modulo1/projeto-blackjack/nossoJogo.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * EXEMPLO DE UTILIZAÇÃO DA 'comprarCarta'
+ * 
+ * 
+    const carta = comprarCarta(); // Sorteia uma carta. Por exemplo, o rei de ouros
+    
+    console.log(carta.texto) // imprime o texto da carta. Exemplo: "K♦️" (indica "K" de ouros)
+    console.log(carta.valor) // imprime o valor da carta (um número). Exemplo: 10 (dado que "K" vale 10)
+ * 
+ * 
+ * 
+ */
+const carta = comprarCarta(); 
+ let usuario
+ let usuario1 
+ let computador
+ let computador1
+ 
+ let i = 0
```

Comment:
Não precisa criar variável de controle de for, fora dele, além disso vale lembrar que essa variável você não utiliza, já que você cria uma com o mesmo nome dentro do próprio for.